### PR TITLE
Merge 6.0.x -> master

### PR DESCRIPTION
**Pull request by Magisus**

### CODE REVIEW

**Comment by justinstoller** [comment]
Looks like I screwed up the merge up from 5.3.x to 6.0.x in https://github.com/puppetlabs/puppetserver/commit/8deb1f485fb871220202b51402e3619ed1ad6475 and this (pre your most recent update) compounds that. Should I open a PR to update the .travis.yml in 6.0.x before we merge this?

**Comment by Magisus** [comment]
Yeah if it's still wrong in 6.0.x we should fix that first and then redo this.

**Comment by Magisus** [comment]
Updated

File: .travis.yml [hunk 1]
```diff
@@ -6,10 +6,22 @@ matrix:
   include:
     - jdk: openjdk11
     - jdk: oraclejdk8
+sudo: required
 script:
   - ./ext/travisci/test.sh
   - $TRAVIS_BUILD_DIR/ext/travisci/secscan.sh "$TRAVIS_BUILD_DIR/src/clj" "clj"
-sudo: required
+matrix:
```
[the remark this answers is not in the document]
Comment (Magisus) [reply .travis.yml]: I dropped the 10 testing we were doing in 6.0.x in favor of 11 here.

**Pull request merged**